test006.c: 函数 `f' :
test006.c:6: エラー: `a' は宣言されていません.
test006.c:9: エラー: `b' は宣言されていません.
test006.c:14: エラー: `c' は宣言されていません.
